internal/cluster/cluster.go

```diff
@@ -178,6 +180,37 @@ func (m *Manager) GetMasterNodesIds(ctx context.Context, c *common.Cluster, db *
 	return m.installationAPI.GetMasterNodesIds(ctx, c, db)
 }
 
+func (m *Manager) tryAssignMachineCidr(cluster *common.Cluster) error {
+	var err error
+	networks := network.GetClusterNetworks(cluster, m.log)
+	if len(networks) == 1 {
```

Comment:
Please add a comment describing why only 1.
I think that if the amount if greater than 1 we want to set a debug log like "More than 1 networks, we don't auto assign machine cidr"

Reply:
The feature is auto assign in case there is a single network.  Otherwise we let the user choose the network.

Reply:
I will add a comment

Reply:
> The feature is auto assign in case there is a single network. Otherwise we let the user choose the network.

I understand that. But for someone who doesn't know the feature and just reads the code - it would be clearer :)

Reply:
OK